src/content/app/tools/blast/views/blast-form/BlastForm.scss

```diff
@@ -36,14 +36,10 @@
 
 .blastFormStepToggle {
   display: flex;
-  justify-content: flex-end;
   gap: 10px;
-  max-width: 1770px; //TODO: To replace with global variables for spaces on the blast form (see ENSWBSITES-1602)
-  margin-left: -2px;
   padding-top: 20px;
   background-color: $white;
   position: sticky;
-  top: 0px;
```

Comment:
This was actually doing an important job


https://user-images.githubusercontent.com/6834224/216003980-1777bef6-a716-4c51-a053-13a74f510beb.mp4

Reply:
Thanks for spotting this. Missed this one while testing.

Reply:
✅